docs/tutorials/scala/mxnet_java_install_and_run_examples.md

```diff
@@ -0,0 +1,121 @@
+# Install and run Java Examples
+
+## Prerequisites:
+Please follow the Step 1 in the [Scala configuration](http://mxnet.incubator.apache.org/install/scala_setup.html#setup-instructions)
+These should help you install the correct Java version and all dependencies.
+
+## Run the Java example project
+We have provided a general MXNet Java template under `scala-package/mxnet-demo/java-demo` which contains complete instruction on running the Hello world and Object detection examples. Please copy the downloaded MXNet Java package jar file to the `java-demo` folder before build the package there.
+
+## Import and run the Java package
+For users using a desktop/laptop, we recommend using IntelliJ IDE as it is tested and supported to provide the necessary documentation for the Java API.
+
+Alternatively, users can follow the second instruction to set up an empty Maven project for Java.
+
+### IntelliJ instruction
+If you are using a computer with Ubuntu16.04 or Mac, you can install IntelliJ to run the Java package. Please follow the instruction below:
+
+1. Create a new Java project in IntelliJ. Fire up IntelliJ and click `Create New Project`.
+
+2. Click `Next`, and in the `Create project from template` window, do not select anything and click `Next` again.
+
+3. In the next window choose your `Project name` and the `Project location` and click on `Finish`.
+
+4. Let's add the Java Inference API jars that we grabbed from Maven Central. At the top of the window, Go to the `File -> Project Structure`. In the popup window that opens up, click on `Libraries -> +` and select the path to the jar files downloaded. Click `Apply` and then click `OK`.
```

Comment:
do the users have to do this? given that the jars will be coming from Maven and they automatically get added.